frontend/test/metabase/scenarios/embedding/embedding-full-app.cy.spec.js

```diff
@@ -135,6 +327,13 @@ describe("scenarios > embedding > full app", () => {
 
       cy.findAllByRole("cell").first().click();
       cy.wait("@getCardQuery");
+
+      // I don't know why this test starts to fail, but this command
+      // will force the cursor to move away from the app bar, if
+      // the cursor is still on the app bar, the logo will not be
+      // be visible, since we'll only see the side bar toggle button.
+      cy.findByRole("button", { name: /Filter/i }).realHover();
```

Comment:
Not sure why this is needed just now. But you could see that without this line, [the test will fail consistently.](https://www.deploysentinel.com/ci/runs/63a5f48af74c60362f669366)

Seems like somehow other changes in this file cause the cursor to pop up in the app bar. (screenshot from Deploysentinel)

<img width="697" alt="image" src="https://user-images.githubusercontent.com/1937582/209404995-3090d2c9-bd26-469a-b8f3-aebb65cd9457.png">